gogs.json

```diff
@@ -0,0 +1,60 @@
+{
+    "homepage": "https://gogs.io/",
+    "description": "A painless self-hosted Git service",
+    "license": "MIT",
+    "version": "0.11.53",
+    "architecture": {
+        "64bit": {
+            "url": "https://cdn.gogs.io/0.11.53/gogs_0.11.53_windows_amd64_mws.zip",
+            "hash": "bd64f8e7ee0f74c6b65bdf2dfa33e7746fab33f4856a07166dce0bd121a48ac8"
+        },
+        "32bit": {
+            "url": "https://cdn.gogs.io/0.11.53/gogs_0.11.53_windows_386_mws.zip",
+            "hash": "6da6a3e9452ea6e0af296c251c491c22bb537525e8cfc340c5e103f8c31acc40"
+        }
+    },
+    "extract_dir": "gogs",
+    "bin": "gogs.exe",
+    "installer": {
+        "script": [
+            "if (!(test-path \"$persist_dir\\custom\\conf\")) { mkdir $dir\\custom\\conf | out-null }",
+            "if (!(test-path \"$persist_dir\\repositories\")) { mkdir $dir\\repositories | out-null }",
+            "if (!(test-path \"$persist_dir\\log\")) { mkdir $dir\\log | out-null }",
+            "if (!(test-path \"$persist_dir\\data\")) { mkdir $dir\\data | out-null }",
+            "if (!(test-path \"$persist_dir\\custom\\conf\\app.ini\")) {",
+            "@\"",
+            "[database]",
+            "DB_TYPE  = sqlite3",
+            "PATH     = $persist_dir\\data\\gogs.db",
+            "",
+            "[repository]",
+            "ROOT = $persist_dir\\repositories",
+            "",
+            "[log]",
+            "ROOT_PATH = $persist_dir\\log",
+            "\"@ | % {$_ -replace \"\\\\\",\"/\"} | out-file -en oem $dir\\custom\\conf\\app.ini }"
```

Comment:
You could go with more readable `-replace '\\\\', '/'`.
No quotes in app.ini around $dir and $persist_dir?? What about paths with spaces?

Reply:
It can also be a bit more compact like [`gitea.json`](https://github.com/lukesampson/scoop-extras/blob/master/gitea.json)

Reply:
@Ash258 gitea works without quotes. Since gitea is a fork of gogs, my guess is that it works too.

![image](https://user-images.githubusercontent.com/432127/45452970-ba0a4e80-b6df-11e8-830d-7c521c1e21c3.png)

Reply:
Missed the $dir quotes, thanks. But I can not find any $persist_dir quotes missing, or do you mean inside the .ini file?

Anyway, replaced the install script with the one from [`gitea.json`](https://github.com/lukesampson/scoop-extras/blob/master/gitea.json). I kept the `persist` in the manifest though, so you can run gogs without the -c.